pytket_qir/qir.py

```diff
@@ -400,11 +463,76 @@ def condition_zero_block():
                 one=lambda: condition_one_block(),
                 zero=lambda: condition_zero_block(),
             )
+        elif isinstance(op, WASMOp):
+            cregs = _retrieve_registers(circ.bits, BitRegister)
+            args = command.args
+
+            inputs: List[str] = []
+            outputs: List[str] = []
+            for reglist, sizes in [
+                (inputs, op.input_widths),
+                (outputs, op.output_widths),
+            ]:
+                for in_width in sizes:
+                    assert in_width > 0
```

Comment:
I think you can have `in_width == 0` since you can define a `WASMOp` for which input and/or output registers are empty. However, I am still a bit confused about that assertion. Do I understand correctly that this assertion must say: " If there are empty registers (which is externally correct in the sense of the caller passing correct arguments), I (in the sense of that current piece of code) should be dealing with it but I am currently not."

Reply:
Done.